### PR TITLE
Update DoL tests

### PR DESCRIPTION
**Pull request by hrfarmer**
Few updates to the DoL tests to use them tomorrow

### CODE REVIEW

**Review by Mikefly123 — APPROVED** [review]
LGTM!

File: PROVESFlightControllerReference/test/day-in-the-life/test_day_in_the_life.py
Comment (ineskhou) [inline PROVESFlightControllerReference/test/day-in-the-life/test_day_in_the_life.py:33]: Maybe have this message in with the Makefile command, since you can't technically run this test suite start to finish

File: PROVESFlightControllerReference/test/day-in-the-life/test_edge_cases_day_in_the_life.py
Comment (ineskhou) [inline PROVESFlightControllerReference/test/day-in-the-life/test_edge_cases_day_in_the_life.py:40]: make usure you run the edge case tests with -s

File: Makefile [hunk 1]
```diff
@@ -442,9 +442,10 @@ gds-integration: framer-plugin
 	@$(GDS_COMMAND) --gui=none --output-unframed-data --uart-device=$(if $(UART_DEVICE),$(UART_DEVICE),/dev/ttyBOARD)
 
 .PHONY: DoL_test
-DoL_test:
+DOL_SERIAL_PORT ?= /dev/ttyACM0
+DoL_test: ## Run the day-in-the-life tests (DOL_SERIAL_PORT=/dev/ttyXXX)
 	@echo "make sure passthrough GDS is running"
-	@$(UV_RUN) pytest test/test_day_in_the_life.py --deployment build-artifacts/zephyr/fprime-zephyr-deployment
+	@DOL_SERIAL_PORT="$(DOL_SERIAL_PORT)" $(UV_RUN) pytest PROVESFlightControllerReference/test/day-in-the-life/test_day_in_the_life.py PROVESFlightControllerReference/test/day-in-the-life/test_edge_cases_day_in_the_life.py --deployment build-artifacts/zephyr/fprime-zephyr-deployment
```
Comment (ineskhou) [inline Makefile:448]: If this is how you want to run the day in the life testing it is up to you, but these tests are meant to be manually checked and understood by the person running them. They are best where they is manual examination after each part for slips and such. My suggesttiong would be to have the day in the life testing be enums and you can choose which test you are running so you can do make DoL_test 1, 2, 3, and its easy for the tester but you can do it slower. Or you could have it so after every test it took user input

**Review by ineskhou — COMMENTED** [review]
LGTM! few comments, but up to you how you want to be running these tests
